tests/CMakeLists.txt

```diff
@@ -1,15 +1,9 @@
 # Setup testing
 enable_testing()
 
-SET(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -isystem ${gtest_SOURCE_DIR}/include")
+#add_compile_options(-g -fsanitize=address)
 
-#SET(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -g")
-#SET(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -fsanitize=address")
-
-SET(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -std=gnu++0x")
-
-configure_file(${CMAKE_SOURCE_DIR}/tests/testfile.dlt ${PROJECT_BINARY_DIR}/tests COPYONLY)
-configure_file(${CMAKE_SOURCE_DIR}/tests/testfilter.txt ${PROJECT_BINARY_DIR}/tests COPYONLY)
```

Comment:
This is also needed for gtest.

Reply:
Fixed by added "add_compile_options(-isystem ${gtest_SOURCE_DIR}/include)" at line 5 in tests/CMakeLists.txt

Reply:
Hi @ssugiura ,
The requested changes have been done.